addons/project/views/project_views.xml

```diff
@@ -887,7 +887,7 @@
                             <field name="partner_phone" widget="phone" attrs="{'invisible': True}"/>
                             <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}"/>
                             <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True}"/>
-                            <field name="recurring_task" attrs="{'invisible': ['|', ('allow_recurring_tasks', '=', False), ('active', '=', False)]}" />
+                            <field name="recurring_task" attrs="{'invisible': ['|', ('allow_recurring_tasks', '=', False), '|', ('active', '=', False), ('parent_id', '!=', False)]}" />
```

Comment:
```suggestion
                            <field name="recurring_task" attrs="{'invisible': ['|', '|', ('allow_recurring_tasks', '=', False), ('active', '=', False), ('parent_id', '!=', False)]}" />
```
This is quite a minor suggestion but I think this is more commonly used and thus should improve readability.